portfolio/src/main/webapp/index.html

```diff
@@ -2,14 +2,17 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Google Step 2020</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>Start</h1>
       <p>This is my portfolio.</p>
+      <p>My name is Seun aka computabeast.</p>
+      <img src = "images/IMG_9177.jpg" style = "height: 100px; width: auto;"/>
```

Comment:
Prefer using css over the style attribute. I suggest giving this image a relatively unique class name.